ocaml/tests/test_tar_ext.ml

```diff
@@ -0,0 +1,162 @@
+(*
+ * Copyright (c) Cloud Software Group, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+open Helpers
+open Tar_ext
+open Bos
+
+let ( // ) = Filename.concat
+
+let gen_test_file_script = "test_data" // "gen_tar_ext_test_file.sh"
+
+let max_size_limit = 2000000L
```

Comment:
There is no link between the limit here and the shell script which generates a file bigger than this size, I think we should at least add some comments to make the relationship clear, to make this test and the shell script easy to maintain later.